build.gradle.kts

```diff
@@ -62,7 +62,7 @@ spotless {
 
         removeUnusedImports()
 
-        palantirJavaFormat(libs.versions.palantir.get()).style("GOOGLE").formatJavadoc(true)
+        palantirJavaFormat(libs.versions.palantir.get()).formatJavadoc(true)
```

Comment:
Is `libs.versions.palantir.get()` the default?  If so, do we need it?

Reply:
I double checked. The default version is based on the following internal "matrix" recommendation:

```
private static final Jvm.Support<String> JVM_SUPPORT = Jvm.<String> support(NAME).add(8, "1.1.0").add(11, "2.28.0").add(21, "2.39.0");
```

So if our Java version is 17, `2.28.0` will be chosen.

I missed such subtlety previously and use the latest version `2.50.0` at `libs.versions.palantir`. Now I am not sure it is a good idea and it seems the default recommended version (i.e. `2.28.0`) is enough?

Do you have more experience on palantir? It is totally new to me. As a starter, I think maybe relying on its internal recommendation based on java version is enough (we could specify version later if we found some defect which was fixed in later version?)?

Reply:
I experimented locally and found if we use default version (`2.28.0` corresonding to `java 11`), the `spotlessJavaDiagnose` task will complain as below:

```
Cannot enable formatJavadoc option, make sure you are using Palantir with version 2.36.0 or later
```

So maybe using the latest version is a good choice?

Reply:
sgtm